### PR TITLE
September 2020 newletter

### PR DESCRIPTION
**Pull request by coolmule0**
Things changed;
- Added timezone info at end of introduction
- October Lunchbytes talk

Things that might not be needed:
- RSE AGM. Not sure if this newletter is the right audience for this event
- October Lunchbytes. I'm not sure of the talk title. I included a call for talks, but is this necessary?

Things still to do:
- PR published a society trustee report. Not sure of the details
- TC did something with TPUs. Not sure of the details

### CODE REVIEW

File: _newsletters/2020-08-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+link:
+description:
+type: text
+---
+
+## University of Sheffield Research Software Engineering Team Newsletter September 2020
+
+This is the 3rd monthly newsletter from the Research Software Engineering Team at Sheffield. We aim to share our experiences and information of other communities for those using software for research.
+This newsletter is a collection of interesting events and opportunities over the coming month. It also includes signposting to other resources that we have found beneficial or interesting.
+You may find the content within interest if you are a Research Technology Professional: someone in research using software, such as researchers, research developers, or people paid to develop software like Research Software Engineers (RSEs).
```
Comment (bobturneruk) [inline _newsletters/2020-08-01-newsletter.md]: I'd include data scientists. I'm not sure about the term "Research Technology Professional" as not all technology is attached to a mouse and keyboard.

File: _newsletters/2020-08-01-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* Last chance at the EPSRC [Call for Access to High Performance Computing (HPC)](https://epsrc.ukri.org/funding/calls/access-to-high-performance-computing/) - 4th September deadline.
+* A new initiative [SORSE](https://sorse.github.io/) was launched in July, with the intent to deliver weekly online content for RSEs to develop skills and collaborate. Abstracts for proposed events can be submitted [here](https://sorse.github.io/programme/call-for-contributions/). Deadline for talk submission is 1st September.
+* [UKRI Coronavirus Hub](https://www.ukri.org/research/coronavirus/) - Collaborations between RSEs and academics can be very productive ways of developing epidemiological models.
+* [Innovation Scholars: Data Science Training in Health & Bioscience](https://esrc.ukri.org/funding/funding-opportunities/innovation-scholars-data-science-training-in-health-bioscience/) - 30th September
+* [EPSRC Doctoral Prize Fellowship 2020](https://www.sheffield.ac.uk/rs/scholarships/calls) - 22nd October
+* [N8 CIR RSE Theme Lead Job](https://n8cir.org.uk/news/n8-cir-are-recruiting/) - 11th September
+
+
+### Web and Blogs
+* Pauls Richmond has made a society trustee report (link?)
```
Comment (bobturneruk) [inline _newsletters/2020-08-01-newsletter.md]: I think @mondus is off today and I can't find this report online. If it doesn't appear in time, I would go without it.

File: _newsletters/2020-08-01-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+* Twin has done something with TPUs.
```
Comment (bobturneruk) [inline _newsletters/2020-08-01-newsletter.md]: @twinkarma have you anything to add?

**Comment by bobturneruk** [comment]
This is looking really good @coolmule0!

File: _newsletters/2020-08-01-newsletter.md (hunk 2 above)
Comment (Robadob) [reply _newsletters/2020-08-01-newsletter.md]: He linked to this on slack: https://society-rse.org/events/agm-2020/ 

That page has:

> Formatted Version of the Trustee Report (http://bit.ly/socrse-report-2020)

**Comment by coolmule0** [comment]
I'm happy with the state of the newsletter now. Thank you for the feedback. The plan would be to send it out Tuesday 1st Sept

**Comment by Robadob** [comment]
@coolmule0 You also need to send the newsletter to `RSE-group@sheffield.ac.uk` (use https://markdown-here.com/ to render the markdown).

(You may have already done this, but I would have expected it to have been approved by now if that were the case).

**Pull request merged**